packages/mdc-textfield/foundation.js

```diff
@@ -167,4 +175,9 @@ export default class MDCTextfieldFoundation extends MDCFoundation {
       badInput: false,
     };
   }
+
+  setValid(isValid) {
+    this.useNativeValidityChecking_ = false;
```

Comment:
I'm not sure if someone would ever use that feature. Is it worth having another public variable to change that ?